Formula/youtube-dl.rb

```diff
@@ -1,18 +1,10 @@
-# Please only update to versions that are published on PyPi as there are too
-# many releases for us to update to every single one:
-# https://pypi.python.org/pypi/youtube_dl
```

Comment:
Why? As I said, all GitHub releases are published to PyPI. https://pypi.org/project/youtube_dl/#history

Plus the fact that we're using GitHub release tarballs, not PyPI source tarballs.